### DISCUSSION TITLE
Gutenberg and Blogging Campaign

### DISCUSSION THREAD

**Discussion by jennimckinnon** [WordPress/Marketing-Team#199] [Ideas]
## Overview: TL;DR
This is an open discussion about launching a marketing campaign focused on promoting [WordPress](https://wordpress.org/), [Gutenberg](https://wordpress.org/plugins/gutenberg/), and blogging for a dedicated month of the year.

## Concept

Similar to November's [National Novel Writing Month](https://nanowrimo.org/) (NaNoWriMo), participants will commit to a month of blogging to reach a goal such as a specific word count or daily blogging streak.

If the goal is reached, participants will receive notable acknowledgement digitally that is also shareable on social media as well as other distribution channels.

## Objective

The marketing campaign will focus on encouraging blogging and its many benefits as well as Gutenberg and the features that make blogging fun, easy, and accessible to all.

The campaign’s goal is to spread more awareness and understanding of Gutenberg, how to use it to inspire creativity and drive action (blogging), promote further adoption as well as an increase in usage and retention rates.

## Framing Device

To make the campaign interesting and engaging for participants, the mental wellness benefits of regular blogging will be highlighted as the main component and reason for the importance of blogging as well as the campaign.

This will help potential participants make an emotional connection that brings on the awareness of the immense positive value blogging can have in their life.

For example, [scientific research confirms](https://www.frontiersin.org/articles/10.3389/fpsyg.2018.02328/full) that in times of uncertainty – such as the historic times the world is in now – people need to be able to make future plans to have the ability to conceive of a future for themselves.

I spoke with [Behavioural Economist Dr. Ali Macintyre](https://alisonmacintyre.com/), and she advises the solution of small, actionable goals. They should have the potential to be completed within a reasonable timeframe as well as give you a sense of hope for the future by connecting yourself to your future.

While the goals are subjective, example goals for during the COVID pandemic could be learning a new language to connect with loved ones better, planning and writing a book, planting a seed daily in your garden, and journaling every day.

Small goals like these practiced consistently and daily compound into larger benefits in your life.

Please note that the campaign wouldn’t focus on the pandemic in an attempt to be evergreen and reusable annually, but it does illustrate the importance and value of blogging as a goal.

## Proposed Strategies and Ideas

[@sereedmedia](https://profiles.wordpress.org/sereedmedia/) suggested in the [marketing channel on Slack](https://wordpress.slack.com/archives/C0GKJ7TFA/p1677257585208899):

- Get the design and development teams involved to make text-based blocks suitable for creative blogging such as quotes
- Get the training team involved to create and promote blogging basics
- Bite-sized TikTok posts or Instagram reels with a hashtag
- Badges for [WordPress.org](https://wordpress.org/) profiles and participants’ sites and blogs
- The name “Augutenberg”

I ([@ninianepress](https://profiles.wordpress.org/ninianepress/)) also mentioned these topic ideas for social media or blog posts for promotion:

- How blogging (and even microblogging) can improve your life, productivity, and mental wellness
- Tutorials, posts about different blocks and formats/types like microblogging or live blogging
- Inspiration by way of sharing what others are blogging about for the duration of the campaign, and/or anecdotes (if we can find them) on how blogging has positively transformed people’s lives or otherwise positively impacted them
- Tips from pro bloggers or people who journal regularly
- A list of unique (and not necessarily well-known) plugins that are fun and useful for bloggin
- Cool plugins for blocks and patterns that are great, fun and useful for blogging
- Updates on new blocks that get developed for the campaign
- Habit-forming tips based on scientific research, and
- Other topics that we can think up

_For the full discussion, please see the [Slack transcript](https://wordpress.slack.com/archives/C0GKJ7TFA/p1677257585208899)_.

### Sub-Campaigns

This is an excellent campaign to potentially get most (if not all) of the WordPress community involved in the sense that WordPress-focused companies can use it to launch their own content and training (sub-campaign) to inspire their customers and clients to participate.

This would open up a whole host of potential for distribution channels as well as virality. The opportunities for this alone are far-reaching as a result, especially when you consider that this campaign is relatable to a much larger global audience.

Companies would benefit from creating a sub-campaign since it would piggyback off the pre-existing and building traction from this campaign, which would offer a statistically much higher chance of their content becoming successful and going viral as a result.

Outreach could be conducted to get as many WordPress-serving companies involved, and perhaps some suggestions and support could be given to help them plan unique campaigns and strategies of their own.

### Potential Media Coverage

Beyond that, the framing of mental wellness could also be an opportunity for potential notable media coverage.

As such, this is an additional area for consideration as someone could head up communication to help ensure suitable coverage.

### Further Participation Potential

Because this campaign would run for an entire month, there’s potential to offer fun-size mini games, campaigns, and gamification in general, that also support the objective.

For example, we can issue weekly challenges for extra acknowledgement, promotion, and kudos. One such challenge could be conceiving of a new type of blogging or block, or other ideas that can promote healthy and collaborative competition, or rather motivation, for the community of participants.

## Challenges

Studies show that it can take anywhere between [18-244 days to form a habit](https://onlinelibrary.wiley.com/doi/abs/10.1002/ejsp.674) or [144 days to develop a new brain map](https://www.amazon.co.uk/Neuroscience-Leadership-Harnessing-Advantage-Business/dp/1137466855). It depends on the individual and their specific situation. As such, it’s tricky to expect the majority of people to start and stick with the campaign throughout the entire month.

This can be solved through training, motivating social media and blog posts as well as gathering and sharing professional experts’ top tips and strategies for forming habits quickly.

The campaign could also stretch across several months: There could be several months of preparatory training and promotion to prime participants for the actual campaign’s main month of blogging. That way, there’s enough time to build the blogging habit to increase the chances of successful engagement throughout the entirety of the campaign.

Another area that could be challenging is coordinating with a long list of companies to encourage creating their own sub-campaign or sharing this one.

A possible solution could be to create a packet that includes suggestions, tips, digital assets, and strategies on how to build a strong sub-campaign.

## Measurables

There are different options in which blogging goals could be chosen and measured. There are a couple areas that could be considered here:

Goals can be suggested, but ultimately up to the participants to choose. Or, one or multiple options could be provided.

As previously touched upon, goals could be set based on word count or daily goals, for example.

But, there could also be goals set up based on values such as democratizing publishing or sharing people’s unique stories and skills that are emotion-based to help people connect more with the campaign, WordPress, and blogging. It will also aid in helping participants’ understanding of the overall positive impact blogging can provide.

No matter what goals end up being decided upon, it will be crucial to consider the user experience. Participants will need an easy way to log and share their progress, and this will require support from teams outside of marketing.

## Support From Other Make Teams

This campaign will need additional support from other Make WordPress teams including, but necessarily limited to:

- Training 
- Documentation
- Core
- Design
- Meta
- Community

## Other Notes and Considerations

I mentioned on Slack that [WordPress.com](https://wordpress.com/) has a great onboarding email series on blogging basics that can be drawn on for inspiration.

@Mark-Andrew (on Slack) suggested checking out [WordPress.com’s Bloganuary](https://bloganuary.wordpress.com/) for inspiration on training for blogging basics.

**Comment by santanainniss** [comment]
This is a phenomenal discussion post, @jennimckinnon! I love how much thought you put into this and all the background info as well. 

One thing that came up for me is that there can be a lot of inertia to get bigger campaigns or issues off the ground in the project. If we can think of the blogging campaign initially as a pilot, we would likely gain more support across the project. Tying it into a smaller campaign to coincide with WP20 in particular would help us:

- capitalize on project momentum
- leverage increased attention from community members
- prove the concept of this project in a high impact, high visibility way
- gain support for an ongoing, larger scale yearly blogging campaign. 

Are you open to collaborating on something like a 20 Blogs for 20 Years campaign?

**Reply by jennimckinnon** [reply]
@santanainniss Thank you!

Excellent points. What we could do is make this the main goal, since it is a larger undertaking. Building any habit, including a writing habit, is tricky. So, we're likely going to want to drag this out, anyway, to prepare people. If we plan on launching this in 2024 or similar, it gives us time to start smaller campaigns that will support this larger one.

If the smaller campaigns were planned to start people on creating a writing habit knowing the main blogging event was next year, it's an opportunity to create a lot of excitement around it that can build over time.

I'm definitely open to collaborating on a "20 Blogs for 20 Years" campaign or similar. I'd absolutely love to do that!

Since the [#WP20 campaign](https://wordpress.org/news/2023/01/wordpress-is-turning-20-lets-celebrate/) is already underway, maybe we can add to that with the intention of it leading up to this campaign.

Maybe have a series of blog posts that help people get more interested in blogging, help them get more inspired to blog, and maybe give some tips, especially around starting to build habits. Part of that series could be posting the top 20 cool blogs for inspiration.

Maybe we could interview some key folks in the WordPress community (and beyond) with some insights about the last 20 years of WordPress. We could think of an angle so that there's useful tips for readers about blogging with Gutenberg, habit-building, or something.

There's definitely lots to go over, but I'm excited to help out in any way that I can.

**Comment by abhansnuk** [comment]
This is a great plan Jenni, and I would add my voice to Santana's that this would be perfect opportunity to align with WordPress's 20th anniversary. There have been lots of discussions on using Gutenberg as a blog campaign over recent years, and this shows the interest and examples that people would value. Will help if I can @jennimckinnon . Wondered also about having a basic blog which meetups that wanted to could add to and distribute locally. I know community is planning some materials / resources for meetups too.

It would be super useful to end users if the campaign can demonstrate the variety of things that can be done with the Block Editor now, which is standard in core (that is the parts of Gutenberg that have been merged).
 
We might be able to get a link with this from the developer blog too on a celebration of how to use from a developer's perspective. I will think more about this and come back to you, but also see what we have coming up in the developer blog that might make a potential link across too.

**Reply by jennimckinnon** [reply]
@abhansnuk Awesome! Love all that. Excellent ideas and thanks for your kind words as well as for sharing.

**Comment by annezazu** [comment]
👋🏼 Hey folks! I love this idea and love seeing this kicked off. If I can help with flagging features to cover, let me know. I might be getting too into the details too quickly but I'd like to make sure we talk about the block editor (or just using blocks or the "editor") rather than "Gutenberg". I think using the term "Gutenberg" is going to cause confusion or make people think they need to use the Gutenberg plugin. A bit more info on that front https://wordpress.org/news/2021/04/become-an-early-adopter-with-the-gutenberg-plugin/

**Reply by DanSoschin** [reply]
Just getting folks to write about why they use WordPress (and then weave in recent features such as blocks, etc.) would be a good way to get started. I agree that using the term "Gutenberg" might be a bit too nuanced for broad audiences.

**Comment by santanainniss** [comment]
I hear two streams of conversation happening in the discussion: WP20 pilot, and a broader blogging project. Both sound like great opportunities! 

I am keen to proceed with a project based on 20 blogs for 20 years of WordPress. I can create a separate and specific GH issue for this, but I'm also happy to collaborate on shaping it. @jennimckinnon How would you feel about collaborating with me to create a GH issue for a 20 blogs/20 years micro campaign to support WP20?

**Comment by sereedmedia** [comment]
@santanainniss and all, before we split this effort or turn it into any one specific thing, can we do a meetup or huddle, on zoom or in the marketing slack channel, to talk this out with the folks who are interested and able to commit some time to working on it? 

there is some real momentum here, courtesy @jennimckinnon's idea and enthusiasm! and we've got @abhansnuk @mark-andrew and @DanSoschin interested and have buy-in from the Site Editor team in the form of @annezazu, all of which is hugely valuable.

I personally would like to find a way for us to work together to make this idea happen.

**Reply by santanainniss** [reply]
I agree that there is great momentum here. I am still hearing two very different streams of conversation. One is based on habit building and writing about specific features of WordPress. The other is based on the Bloganuary model and tying that to WP20, i.e., 20 blogging prompts for existing bloggers to blog about in the 20 days leading up to the 20th anniversary. In my mind, those are quite different in scale and goal. I don't view moving forward with 20 blogs/20 years as splitting efforts. 

If folks are not interested in contributing to 20 blogs/20 years, I can happily disagree and commit.

**Comment by jaapwiering** [comment]
Just dropping an idea.
20 years WordPress = from blogs to blocks

**Comment by femkreations** [comment]
Hi folks. Great initiative by @jennimckinnon. I love the idea of 20 blogs for 20 years to support WP20 which will lead to the bigger campaign. Happy to participate and talk about the various features of the block editor that makes the writing flow easy for bloggers.

**Comment by santanainniss** [comment]
Update: 

Last week, several contributors met synchronously to discuss the 20 Blogs for WP20 initiative: Nyasha G., @sereedmedia, @jennimckinnon, Mark Andrew, and myself. 

We discussed:

-20 blogs for 20 years is a great micro-campaign initiative to test the waters with
- 20 blogs in the 20 day lead up to May 27
- It's a tight timeline, but with many eyes all campaigns are possible! (see what I did there? ;))
- This is a contributor-led WP20 campaign!
- This is a small proof-of-concept initiative before moving on to explore a broader Gutenberg blogging initiative 
- Next steps: creating a ticket for this initiative, creating an announcement post w/ call for contribution, contribution to brainstorm and select 20 blogging prompts, collaborating for promoting the initiative and 
- KPIs: engagement!!!! Number of blogs written, participation by contributors, reception in the community
-Goals: celebrate the wonder and magic of WordPress over the last two decades, celebrating WP from Blogs to Blocks (HT @jaapwiering for that 🔥), engaging the community, reflecting on 20 years of WP

Timeline goals:

- Ticket created by 14 April
- Make blog by 18 April
- Contribution to blog prompt brainstorming + selection: 18 April - 25 April
- Finalizing prompt selection: 25 April - 2 May
- Campaign launch: 7 May
- Community engagement, promotion, and blog writing: 5 May to 27 May


Immediate next steps:

-Draft GH ticket @sereedmedia + @santanainniss 
-Draft Make Blog post: @sereedmedia + @santanainniss

**Reply by jaapwiering** [reply]
Happy to see a simple text idea being launched!

**Comment by sereedmedia** [comment]
Oh *wow* to From Blogs to Blocks!!! I loooveeee it @jaapwiering !!!!

**Reply by nomad-skateboarding-dev** [reply]
Agreed! Excited for this!
